plugin/Renderer/InputFieldCheckboxRenderer.php

```diff
@@ -51,11 +51,18 @@ class InputFieldCheckboxRenderer
 	 *
 	 * @param string $name
 	 * @param mixed $value
-	 *
+	 * @param null $_pContainer
+	 * @throws Exception
 	 */
-	public function __construct($name, $value)
+	public function __construct($name, $value, $_pContainer = null)
 	{
-		parent::__construct('checkbox', $name, $value);
+		if (empty($_pContainer)) {
```

Comment:
Could you please turn `empty` into `!== null`? I would appreciate if `$_pContainer` would get a type hint.